qcengine/tests/test_harness_canonical.py

```diff
@@ -24,6 +24,9 @@
     ("torchani", {"method": "ANI1x"}),
     ("turbomole", {"method": "pbe", "basis": "6-31G"}),
     ("xtb", {"method": "GFN2-xTB"}),
+    ("adcc", {"method": "adc2", "basis": "6-31G"}),
+    ("gcp", {"method": "hf3c"}),
+    ("mrchem", {"method": "blyp"}),
```

Comment:
> From https://mrchem.readthedocs.io/en/latest/users/user_ref.html, I gather this is intended behavior. Is a default out of the question so that a plain molecule+model can run?

That parameter is (roughly) equivalent to the basis set for GTO software, so a default wouldn't make sense. What about the following?

```suggestion
    ("mrchem", {"method": "blyp",  "keywords": {"world_prec": 1.0e-3}}),
```